docs/tutorials/python/json_schema.md

```diff
@@ -0,0 +1,124 @@
+3. Bind the JSON Schema to a folder
+4. Add annotations and validate against the schema
+5. Attach and validate a file
+6. View schema validation statistics and results
+
+## Prerequisites
+* You have a working [installation](../installation.md)of the Synapse Python Client.
+* Make sure that you have completed the [Project](./project.md) tutorial, which covers creating and managing projects in Synapse. This is a prerequisite because you need a project to organize and store the folder used in this tutorial.
+* You are familiar with Synapse concepts: [Project](./project.md), [Folder](./folder.md), [File](./file.md).
+* You are familiar with [adding annotations](./annotation.md) to synapse entity.
+
+
+## Step 1: Set up Synapse Python client and retrieve project
+
+```python
+{!docs/tutorials/python/tutorial_scripts/json_schema.py!lines=1-20}
+```
+
+## Step 2: Take a look at the constants and structure of the JSON schema
+
+For the purpose of this tutorial, an organization named "myUniqueAlzheimersResearchOrgTutorial" has been created, and within it, a schema named clinicalObservations has been registered.
+
+```python
+{!docs/tutorials/python/tutorial_scripts/json_schema.py!lines=23-49}
+```
+
+Derived annotations allow you to define default values for annotations based on schema rules, ensuring consistency and reducing manual input errors. As you can see here, you could use derived annotations to prescribe default annotation values. Please read more about derived annotations [here](https://help.synapse.org/docs/JSON-Schemas.3107291536.html#JSONSchemas-DerivedAnnotations).
+
+
+## Step 3: Retrieve test organization
+Next, retrieve an organization:
+```python
+{!docs/tutorials/python/tutorial_scripts/json_schema.py!lines=51-54}
+```
+
+## Step 4: Bind the JSON schema to the folder
+After retrieving the organization, you can now bind your json schema to a test folder. When you bind a JSON Schema to a project or folder, then all items inside of the project or folder will inherit the schema binding, unless the item has a schema bound to itself.
+
+When you bind the schema, you may also include the boolean property `enable_derived_annos` to have Synapse automatically calculate derived annotations based on the schema:
+
+```python
+{!docs/tutorials/python/tutorial_scripts/json_schema.py!lines=56-63}
+```
+
+## Step 5: Retrieve the Bound Schema
+Next, we can retrieve the bound schema:
+```python
+{!docs/tutorials/python/tutorial_scripts/json_schema.py!lines=65-67}
+```
+
+## Step 6: Add Invalid Annotations to the Folder and Store
+Try adding invalid annotations to your folder: This step and the step below demonstrate how the system handles invalid annotations and how the schema validation process works.
+```python
+{!docs/tutorials/python/tutorial_scripts/json_schema.py!lines=70-76}
+```
+
+In the synapse web UI, you could also see your invalid annotations being marked by a yellow label similar to this:
```

Comment:
Is an error message or something else printed to the console when there is an issue with the annotations you set?


I also see that `Step 7: Validate Folder Against the Schema` might cover this, however, I would expect that we mention the programmatic way to verify this data first, followed by the UI way to verify this. I would also combine these into the same step.

Lastly - if there are messages that should be printing to the console that the user would be expecting to see, that should be added to the tutorial. For example in the file tutorial: https://github.com/Sage-Bionetworks/synapsePythonClient/blob/d9998ef905aa98a40ed98a5590df4bd314c51ccb/docs/tutorials/python/file.md?plain=1#L105-L121